assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -20,28 +20,93 @@ import ProductLowStockBadge from './product-low-stock-badge';
 /**
  * Cart line item table row component.
  */
-const CartLineItemRow = ( { lineItem = {} } ) => {
+const CartLineItemRow = ( {
+	lineItem = {
+		key: '',
+		id: 0,
+		quantity: 0,
+		name: '',
+		summary: '',
+		short_description: '',
+		sku: '',
+		remaining_stock: null,
+		low_stock_remaining: null,
+		low_stock_threshold: null,
+		sold_individually: false,
+		permalink: '',
+		images: [],
+		variation: [],
+		prices: {
+			currency_code: 'US',
+			currency_minor_unit: 2,
+			currency_symbol: '$',
+			currency_prefix: '$',
+			currency_suffix: '',
+			currency_decimal_separator: '.',
+			currency_thousand_separator: ',',
+			price: '0',
+			regular_price: '0',
+			sale_price: '0',
+			price_range: null,
+		},
+		totals: {
+			currency_code: 'US',
+			currency_minor_unit: 2,
+			currency_symbol: '$',
+			currency_prefix: '$',
+			currency_suffix: '',
+			currency_decimal_separator: '.',
+			currency_thousand_separator: ',',
+			line_subtotal: '0',
+			line_subtotal_tax: '0',
+			line_total: '0',
+			line_total_tax: '0',
+		},
+	},
+} ) => {
+	/**
+	 * @type {import('@woocommerce/type-defs/cart').CartItem}
```

Comment:
What is best practice for importing type-defs? Top of file? Inline? Could use a coding standards doc on type-defs specifically.

Reply:
Oh ya I missed this, I thought it would import at the top of the file. This is accepted (and works) but I think I'd prefer at the top of the file along with the other imports so at a glance we can see all that the code in that file depends on. I'll fix this instance.

> Could use a coding standards doc on type-defs specifically.

Yup I have a personal @todo to both document type-defs usage (with typescript) and implement some sort of standard to start with (maybe add linting rule too).